### ISSUE TITLE
source_id registration of CNRM-ALADIN64E1

### ISSUE THREAD

**Issue by pierrenabat** [WCRP-CORDEX/cordex-cmip6-cv#89]
### label

CNRM-ALADIN64E1

### label_extended

CNRM-ALADIN regional climate model, version 6.4 E1

### source_id

CNRM-ALADIN64E1

### source_type

ARCM

### release_year

2023

### activity_participation

DD

### institution_id

CNRM-MF

### further_info_url

https://www.umr-cnrm.fr/spip.php?article1135

### What license do you choose?

CC BY 4.0

**Comment by jesusff** [comment]
Thank you for the input. The `further_info_url` points to a quite generic website about all CNRM models. Shall we point it to the specific one for CNRM-ALADIN (http://www.umr-cnrm.fr/spip.php?article125&lang=en)?

**Comment by pierrenabat** [comment]
Yes you can change it: it will be better to point to this page of CNRM-ALADIN. It will be soon updated to include the version 6.4 E1

**Comment by jesusff** [comment]
Hi Pierre @pierrenabat 

just to confirm that the name of the model will include the CNRM part. Note that the `institution_id` in the CORDEX-CMIP6 DRS goes in a separate element, so your files will be named e.g. `..._CNRM_CNRM-ALADIN64E1_...`, right? This is different from your convention in CORDEX-CMIP5.
If you agree, then I'll update the model name in the [simulation status page](https://wcrp-cordex.github.io/simulation-status/CMIP6_downscaling_plans.html?search.search=aladin64e1). Please, check that the simulations look otherwise fine there, or open an issue/PR for corrections.

**Comment by pierrenabat** [comment]
Hi Jesus,
yes we would like to keep CNRM in the name of the model itself : CNRM-ALADIN64E1.
We have seen the change from the convention in CORDEX-CMIP5, and our files will be named as :
tas_EUR-12_CNRM-ESM2-1_ssp585_r1i1p1f2_CNRM-MF_CNRM-ALADIN64E1_v1-r1_mon_204101-205012.nc
Thanks for updating the simulation status page

**Comment by gnikulin** [comment]
looks fine
